packages/cms-cli/commands/secrets.js

```diff
@@ -95,8 +100,39 @@ function configureSecretsDeleteCommand(program) {
   addConfigOptions(program);
 }
 
+function configureSecretsListCommand(program) {
+  program
+    .version(version)
+    .description('List all HubSpot secrets')
+    .action(async (options = {}) => {
+      setLogLevel(options);
+      logDebugInfo(options);
+      const { config: configPath } = options;
+      loadConfig(configPath);
+
+      if (!(validateConfig() && (await validatePortal(options)))) {
+        process.exit(1);
+      }
+      const portalId = getPortalId(options);
+
+      try {
+        const { results } = await getSecrets(portalId);
+        logger.log(`Secrets for portal: ${portalId}:`);
+        results.forEach(secret => logger.log(secret));
```

Comment:
It might present better if we grouped.

```javascript
const groupLabel = `Secrets for portal: ${portalId}:`;
logger.group(groupLabel);
/* ...log each secret... */
logger.groupEnd(groupLabel);
```